openmmtools/integrators.py

```diff
@@ -35,26 +35,93 @@
 # GLOBAL IMPORTS
 # ============================================================================================
 
-import numpy
+import numpy as np
 import logging
 import re
 
 import simtk.unit
 
-import simtk.unit as units
+import simtk.unit as unit
 import simtk.openmm as mm
 
 from openmmtools.constants import kB
 from openmmtools import respa
 
 logger = logging.getLogger(__name__)
 
+# Energy unit used by OpenMM unit system
+_OPENMM_ENERGY_UNIT = unit.kilojoules_per_mole
 
 # ============================================================================================
 # BASE CLASSES
 # ============================================================================================
 
-class RestorableIntegrator(mm.CustomIntegrator):
+class PrettyPrintableIntegrator(object):
```

Comment:
Nice! This will be helpful in verifying that complicated CustomIntegrators are being constructed as intended